database/models/inventory.model.js

```diff
@@ -0,0 +1,32 @@
+import mongoose, { Types } from "mongoose";
+const schema = new mongoose.Schema({
+    quantity:{
+        type: Number,
+        required: true,
+    },
+
+    stock: {
+        type: Boolean,
+        required: true,
+    },
+
+    pharmacy: {
+        type:Types.ObjectId,
+        ref: 'pharmacy', 
+        required: true
+    },
+
+    pharmacist: {
+        type:Types.ObjectId,
+        ref: 'Pharmacist', 
+        required: true
+    },
+
+    updateDate:{
```

Comment:
delete updateDate (we do not need it because we have timestamps:true (same as update)